qiskit/quantum_info/operators/symplectic/clifford.py

```diff
@@ -110,93 +118,245 @@ def __init__(self, data, validate=True):
-        return StabilizerTable(self._table[self.num_qubits : 2 * self.num_qubits])
+        return StabilizerTable(self.table[self.num_qubits : 2 * self.num_qubits])
 
     @stabilizer.setter
+    @deprecate_function(
+        "The Clifford.stabilizer method is deprecated as of Qiskit Terra 0.19.0 "
+        "and will be removed no sooner than 3 months after the releasedate. "
+        "Use Clifford.paulis method instead.",  # pylint:disable=bad-docstring-quotes
+    )
     def stabilizer(self, value):
         """Set the value of stabilizer block of the StabilizerTable"""
         inds = slice(self.num_qubits, 2 * self.num_qubits)
-        self._table.__setitem__(inds, value)
+        self._paulis.__setitem__(inds, value)
 
     @property
+    @deprecate_function(
+        "The Clifford.destabilizer method is deprecated as of Qiskit Terra 0.19.0 "
+        "and will be removed no sooner than 3 months after the releasedate. "
+        "Use Clifford.destabilizer_tableau or Clifford.paulis method instead.",  # pylint:disable=bad-docstring-quotes
+    )
     def destabilizer(self):
         """Return the destabilizer block of the StabilizerTable."""
-        return StabilizerTable(self._table[0 : self.num_qubits])
+        return StabilizerTable(self.table[0 : self.num_qubits])
 
     @destabilizer.setter
+    @deprecate_function(
+        "The Clifford.destabilizer method is deprecated as of Qiskit Terra 0.19.0 "
+        "and will be removed no sooner than 3 months after the releasedate. "
+        "Use Clifford.paulis method instead.",  # pylint:disable=bad-docstring-quotes
+    )
     def destabilizer(self, value):
         """Set the value of destabilizer block of the StabilizerTable"""
         inds = slice(0, self.num_qubits)
-        self._table.__setitem__(inds, value)
+        self._paulis.__setitem__(inds, value)
+
+    def tableau(self, x=True, z=True, stabilizer=True, destabilizer=True):
+        """Return the tableau of the Clifford.
+
+        Args:
+            x (bool): if True, it returns x component.
+            z (bool): if True, it returns z component.
+            stabilizer (bool): if True, it returns stablizer component.
+            destabilizer (bool): if True, it returns destabilizer component.
+
+        Returns:
+            np.ndarray: the tableau.
+
+        """
+        if stabilizer and destabilizer:
+            indices = slice(None)
+        elif stabilizer:
+            indices = slice(self.num_qubits, 2 * self.num_qubits)
+        elif destabilizer:
+            indices = slice(0, self.num_qubits)
+        else:
+            return np.array([], dtype=bool)
+
+        if x and z:
+            return np.hstack((self.paulis.x[indices], self.paulis.z[indices]))
+        elif x:
+            return self.paulis.x[indices].copy()
+        elif z:
+            return self.paulis.z[indices].copy()
+        else:
+            return np.array([], dtype=bool)
+
+    def stabilizer_tableau(self):
```

Comment:
If they aren't currently used I don't think it's necessary to add the stab/destab/x/z version, since you can do it with kwargs of the `tableau` function (and update the deprecation notes accordingly for the ones that refer to these)

Another idea is you could also add a `tableau` method to `PauliList` and then these could also be invoked via `clifford.stabilzier_list().tableau()`.

Reply:
Nice idea. I adopted the latter one.

Reply:
`clifford.stabilziers().tableau()`